### PR TITLE
Added Lizard styling for pump stations (SVG)

### PR DESCRIPTION
**Pull request by GeOdin**

### CODE REVIEW

File: lizard_downloader.py [hunk 1]
```diff
@@ -216,7 +217,7 @@ def show_data(self):
         """ Show the data as a new layer on the map """
 
         # Setup
-        WGS84 = "EPSG:4326"
+        wgs84 = "EPSG:4326"
```
Comment (arjenvrielink) [inline lizard_downloader.py]: why is this changed to lower_case? It's a constant so upper case is better I think.

File: lizard_downloader.py [hunk 2]
```diff
@@ -230,7 +231,17 @@ def show_data(self):
 
         # Create a new memory vector layer
         self.layer = QgsVectorLayer(
-            "{}?crs={}".format(geometry_type, WGS84), asset_types[0], "memory")
+            "{}?crs={}".format(geometry_type, wgs84), asset_types[0], "memory")
+
+        # Add Lizard style (SVG)
```
Comment (arjenvrielink) [inline lizard_downloader.py]: I would refactor this code block into a function and move it to a `utils` module. Then import it and call it here. That way other modules can also use this general functionality.

Write a function called `apply_style(layer, asset_type)` in a module `styler` in the `utils` module, much like the 3di qgis plugin does. Pass a layer and an asset_type et voila!

**Comment by arjenvrielink** [comment]
in general: looks fine, just refactor the styling code block to a function in a utils module and I think upper case is better for `WGS84` though if you disagree, that's fine with me as well.

File: lizard_downloader.py (hunk 1 above)
Comment (GeOdin) [reply lizard_downloader.py]: I changed it into lower case, because that would be more in accordance with PEP8.
But if upper case is the convention for constants, I'll change it.

Comment (jackieleng) [reply lizard_downloader.py]: Yeah it's pep8: https://www.python.org/dev/peps/pep-0008/#constants

Comment (GeOdin) [reply lizard_downloader.py]: My bad, my version of flake8 did not check for constants. 
I'll change WGS84 back to upper case.

File: utils/get_data.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+"""
+
+# Import
```
Comment (jackieleng) [inline utils/get_data.py]: ``# Import`` is redundant
License information is also redundant (already in the LICENSE file in the project root).

 I think it's better to remove these, and just have a docstring that describes the module.

File: utils/layer.py [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+"""
+
+# Import
```
Comment (jackieleng) [inline utils/layer.py]: see above comment above import and license information

File: utils/constants.py [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+"""
```
Comment (jackieleng) [inline utils/constants.py]: redundant license info (see below)

File: utils/styler.py [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+"""
+
+# Import
```
Comment (jackieleng) [inline utils/styler.py]: see above comment above import and license information

File: utils/layer.py [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+from PyQt4.QtCore import QVariant
+from qgis.core import QgsFeature
+from qgis.core import QgsField
+from qgis.core import QgsGeometry
+from qgis.core import QgsMapLayerRegistry
+from qgis.core import QgsPoint
+from qgis.core import QgsVectorLayer
+
+from constants import ASSET_GEOMETRY_TYPES
```
Comment (jackieleng) [inline utils/layer.py]: should be:

```python
from .constants import .....
from .styler import .....
```

right?
